common/forking/gasSchedule.go

```diff
@@ -131,7 +131,9 @@ func (g *gasScheduleNotifier) EpochConfirmed(epoch uint32, _ uint64) {
 
 	g.lastGasSchedule = newGasSchedule
 	for _, handler := range g.handlers {
-		handler.GasScheduleChange(g.lastGasSchedule)
+		if !check.IfNil(handler) {
```

Comment:
this is a safe check but a nil handler could never be added in the handlers as the RegisterNotifyHandler function has already a nil check for each handler it adds